docs/manuals/source/Appendix/Debugging.rst

```diff
@@ -158,26 +158,60 @@ Installing debug symbols packages
 .. index::
    single: debug symbols package; dbg; debuginfo; debug
 
-Our Linux packages are stripped of debugging symbols, so you need an extra step to install their rpm debuginfo or deb dbg equivalent
+Our Linux packages are stripped of debugging symbols, so you need an extra step to install their rpm debuginfo or deb dbg equivalent.
```

Comment:
```suggestion
Our binaries do not contain debug symbols, but as they are needed for proper debugging, we package the debug symbols separately.
```

Those symbols are almost always called `debug symbols` (at least i have never heard `debugging symbols` before).  Im not sure about the other changes, what do you think ?

Reply:
done